### PR TITLE
Shawnrecinto factors question keys

### PR DESCRIPTION
**Pull request by motikagan-okta**
## Description:
- **What's changed?** Recreating pull request https://github.com/okta/okta-developer-docs/pull/421 to resolve merge conflicts caused by docs file structure migration.
- **Is this PR related to a Monolith release?** No.

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 1]
```diff
@@ -3017,6 +3017,33 @@ Specifies the profile for a `question` factor
 }
 ```
 
+#### Question Profile - Unique Keys for question
+
+Referenced from [okta-auth-js questions](https://github.com/okta/okta-auth-js/blob/master/test/xhr/questions.js)
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md]: Let's drop this in favor of something like:
> The keys for the built-in security questions are listed below.

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 2]
```diff
@@ -3017,6 +3017,33 @@ Specifies the profile for a `question` factor
 }
 ```
 
+#### Question Profile - Unique Keys for question
```
Comment (nbarbettini) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md]: Can you clean up this title? Maybe `Built-in Question Keys`

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md (hunk 1 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/reference/api/factors/index.md]: Made the edit.

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md (hunk 2 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/reference/api/factors/index.md]: Edited to use suggested wording.

**Comment by motikagan-okta** [comment]
@nbarbettini, could you take another look at this, and approve if okay?

**Pull request merged**